common/src/main/java/org/opensearch/sql/common/utils/StringUtils.java

```diff
@@ -107,6 +107,6 @@ private static boolean isQuoted(String text, String mark) {
   }
 
   public static String removeParenthesis(String qualifier) {
-    return qualifier.replaceAll("\\[.+\\]", "");
+    return qualifier.replaceAll("\\[\\d+\\]", "");
```

Comment:
Is it a regex? Do you want to match `:` or `\d+:\d+` (or even `\d*:\d*`)?
Add a comment for this function please.